diff --git a/VGP/HTSLIB/Makefile b/VGP/HTSLIB/Makefile
index 45e3683..deed325 100755
--- a/VGP/HTSLIB/Makefile
+++ b/VGP/HTSLIB/Makefile
@@ -22,9 +22,9 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
-CC     = gcc
-AR     = ar
-RANLIB = ranlib
+CC     ?= gcc
+AR     ?= ar
+RANLIB ?= ranlib
 
 # Default libraries to link if configure is not used
 htslib_default_libs = -lz -lm -lbz2 -llzma -lcurl
diff --git a/VGP/Makefile b/VGP/Makefile
index 87a5f7e..185f234 100644
--- a/VGP/Makefile
+++ b/VGP/Makefile
@@ -12,6 +12,8 @@ ONE_DPND = $(ONE_LIB) ../Core/ONElib.h VGPschema.h
 GENE_CORE = gene_core.c LIBDEFLATE/libdeflate.a
 GENE_DPND = $(GENE_CORE) gene_core.h
 
+CC ?= gcc
+
 include HTSLIB/htslib_static.mk
 
 deflate.lib: LIBDEFLATE
@@ -21,25 +23,25 @@ libhts.a: HTSLIB
 	cd HTSLIB; make; cd ..
 
 VGPzip: VGPzip.c $(GENE_DPND)
-	gcc $(CFLAGS) -o VGPzip VGPzip.c $(GENE_CORE) -lpthread
+	$(CC) $(CFLAGS) -o VGPzip VGPzip.c $(GENE_CORE) -lpthread
 
 VGPseq: VGPseq.c $(ONE_DPND) $(GENE_DPND) HTSLIB/libhts.a
-	gcc $(CFLAGS) -o VGPseq -I./HTSLIB $(HTSLIB_static_LDFLAGS) VGPseq.c $(ONE_LIB) $(GENE_CORE) HTSLIB/libhts.a -lpthread $(HTSLIB_static_LIBS)
+	$(CC) $(CFLAGS) -o VGPseq -I./HTSLIB $(HTSLIB_static_LDFLAGS) VGPseq.c $(ONE_LIB) $(GENE_CORE) HTSLIB/libhts.a -lpthread $(HTSLIB_static_LIBS)
 
 VGPpair: VGPpair.c $(ONE_DPND) $(GENE_DPND)
-	gcc $(CFLAGS) -o VGPpair VGPpair.c $(ONE_LIB) $(GENE_CORE) -lpthread
+	$(CC) $(CFLAGS) -o VGPpair VGPpair.c $(ONE_LIB) $(GENE_CORE) -lpthread
 
 VGPpacbio: VGPpacbio.c pb_expr.c pb_expr.h $(ONE_DPND) $(GENE_DPND)
-	gcc $(CFLAGS) -o VGPpacbio VGPpacbio.c pb_expr.c $(ONE_LIB) $(GENE_CORE) -lpthread
+	$(CC) $(CFLAGS) -o VGPpacbio VGPpacbio.c pb_expr.c $(ONE_LIB) $(GENE_CORE) -lpthread
 
 VGPcloud: VGPcloud.c msd.sort.c msd.sort.h $(ONE_DPND) $(GENE_DPND)
-	gcc $(CFLAGS) -o VGPcloud VGPcloud.c msd.sort.c $(ONE_LIB) $(GENE_CORE) -lpthread
+	$(CC) $(CFLAGS) -o VGPcloud VGPcloud.c msd.sort.c $(ONE_LIB) $(GENE_CORE) -lpthread
 
 Dazz2pbr: Dazz2pbr.c $(ONE_DPND) $(GENE_DPND)
-	gcc $(CFLAGS) -o Dazz2pbr Dazz2pbr.c $(ONE_LIB) $(GENE_CORE) -lpthread
+	$(CC) $(CFLAGS) -o Dazz2pbr Dazz2pbr.c $(ONE_LIB) $(GENE_CORE) -lpthread
 
 Dazz2sxs: Dazz2sxs.c $(ONE_DPND) $(GENE_DPND)
-	gcc $(CFLAGS) -o Dazz2sxs Dazz2sxs.c $(ONE_LIB) $(GENE_CORE) -lpthread
+	$(CC) $(CFLAGS) -o Dazz2sxs Dazz2sxs.c $(ONE_LIB) $(GENE_CORE) -lpthread
 
 tidyup:
 	rm -f $(ALL)
